src/aditiGJUN20/ExtractValueFromInput.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Extract ItemId from below input
+ * String input = https://www.facebook.com/technocredits/itemId=12345/
+ * String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+ * String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+ * String input = https://www.facebook.com/technocredits/empid=123/
+ * 
+ * Output : 
+ * item id is 12345 
+ *      OR
+ *      no item id present.
+ */
+
+package aditiGJUN20;
+
+import java.util.Arrays;
+
+public class ExtractValueFromInput {
+	void findValueOf(String url, String findValueOf) {
+		String itemId = "";
+		if (url.contains(findValueOf)) {
+			System.out.println("Input Url: " + url);
+			String tempStr = url.substring(url.lastIndexOf(findValueOf));
+			for (int index = 0; index < tempStr.length(); index++) {
+				if (String.valueOf(tempStr.charAt(index)).equals("&")
+						|| String.valueOf(tempStr.charAt(index)).equals("/")) {
+					break;
```

Comment:
Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.